applications/mne_analyze/plugins/view3d/view3d.cpp

```diff
@@ -353,3 +367,53 @@ void View3D::onModelChanged(QSharedPointer<ANSHAREDLIB::AbstractModel> pNewModel
         newDipoleFit(qSharedPointerCast<DipoleFitModel>(pNewModel)->data(QModelIndex()).value<INVERSELIB::ECDSet>());
     }
 }
+
+//=============================================================================================================
+
+void View3D::onModelRemoved(QSharedPointer<ANSHAREDLIB::AbstractModel> pRemovedModel)
+{
+    if(pRemovedModel->getType() == MODEL_TYPE::ANSHAREDLIB_DIPOLEFIT_MODEL) {
```

Comment:
will this delete all dipole model 3D items or just the one for the current subject -> session -> functional data

Reply:
Just the currently selected one being deleted, the only one shown in the 3d view is the one selected in the data manger, it only seems to support showing one at a time.